src/main/steps/applicant2/needs-to-confirm-joint-application/content.ts

```diff
@@ -0,0 +1,26 @@
+import dayjs from 'dayjs';
+import advancedFormat from 'dayjs/plugin/advancedFormat';
+
+import { TranslationFn } from '../../../app/controller/GetController';
+import type { CommonContent } from '../../common/common.content';
+
+dayjs.extend(advancedFormat);
+
+const en = ({ partner, formState }: CommonContent) => ({
+  title: `Your ${partner} needs to confirm your joint application`,
+  line1: `Your ${partner} needs to confirm your joint application. They have been sent an email inviting them to review your combined answers and confirm the application. They should do this by `,
+  reviewDate: dayjs().add(2, 'weeks').format('MMMM Do YYYY'),
```

Comment:
use the dueDate

Reply:
not related to this ticket but i tihnk we might need to replace this dayjs to duedate in other places.